recipes/r-text2vec/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = '0.5.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-text2vec
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: text2vec_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/text2vec_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/text2vec/text2vec_{{ version }}.tar.gz
+  sha256: ecf97ef61578f69d5cb6691385a01ed10080aa2914b54d4671abbffb2a28976d
+
+build:
+  number: 0
+  # skip osx and win builds because of issue with r-rcppparallel
+  # the OSX error seems to be related to C++11
+  skip: true  # [osx or win]
```

Comment:
Maybe spelling this as "True" fixes the error, although I would have expected the LINT checks to complain of it was a problem.